package.json

```diff
@@ -115,6 +115,9 @@
       "views/**/*"
     ]
   },
+  "prettier": {
+    "trailingComma": "none"
+  },
```

Comment:
I'm surpised you didn't put this in it's own file.

Reply:
Yeah me as well, I guess Im changing opinion when I see the amount of folders and files growing and growing XD